home_work_8/writing_to_files/csv_converter.py

```diff
@@ -0,0 +1,14 @@
+"""
+The module takes a list of dictionaries and writes it to a csv file
+"""
+
+import csv
+
+
+def create_csv(data, file):
+    with open(file, 'w', encoding='utf-8') as f:
```

Comment:
Добавляйте параметр `newline`, xтобы в csv файл запись происходила в каждую строку в каждую строку.
```suggestion
    with open(file, 'w', newline='', encoding='utf-8') as f:
```